regression/pages/whitelabel/activate_account.py

```diff
@@ -0,0 +1,51 @@
+"""
+Page for account activation
+"""
+
+from bok_choy.page_object import PageObject
+
+
+class ActivateAccount(PageObject):
+    """
+    Activate Account Class
+    """
+
+    def __init__(self, browser, activation_url):
+        """
+        Activate account url has to be set by the test
+        """
+        super(ActivateAccount, self).__init__(browser)
+        self.activate_account_url = activation_url
+
+    @property
+    def url(self):
+        """
+        Construct a URL to the page
+        """
+        return self.activate_account_url
+
+    def is_browser_on_page(self):
+        """
+        Is browser on the page?
+        Returns:
+            True if message element is present:
+        """
+        return self.q(css='.message>.valid').present
+
+    def is_account_activation_complete(self):
+        """
+        Is account activation complet?
+        Returns:
+            True if activation complete message is visible:
+        """
+        return self.q(
+            css='.message>.valid'
+        ).filter(lambda elem: elem.text == 'Activation Complete!').visible
+
+    def go_to_dashboard_after_activation(self):
```

Comment:
this should just be `go_to_dashboard`, shouldn't it? Someone might call this method regardless of whether or not they just activated someone.